### DISCUSSION TITLE
Is it possible to make the window border rounded?

### DISCUSSION THREAD

**Discussion by ccrsxx** [tomasklaen/uosc#74] [Q&A] [answered]
I couldn't find the option in the config file. Is it possible or not to make it rounded?

**Comment by tomasklaen** [comment, marked as answer]
No, that's not something uosc can control. That's in the hands of either mpv, or your desktop environment.

**Reply by ccrsxx** [reply]
I'm using Windows 11, nearly all apps have rounded corners. Even MPV itself by default is rounded, but when I use this theme the rounded corner is gone. I think there's something in this theme that prevents the border to be rounded by default. Do you think there's an option that might've disabled the OS to round the border?

**Reply by tomasklaen** [reply]
I think `border=no` does that. It switches mpv's window to be just the content outside of any desktop environment decorations.

So removing that config, or setting it to `=yes`, should give you rounded corners, but than you also get a titlebar.

**Reply by ccrsxx** [reply]
Thanks for the response. I'll check it out tomorrow.
